packages/adapter-node/src/server.js

```diff
@@ -40,12 +40,20 @@ export function createServer({ render }) {
 		prerendered_handler,
 		async (req, res) => {
 			const parsed = new URL(req.url || '', 'http://localhost');
+
+			try {
+				var body = await getRawBody(req); // eslint-disable-line no-var
```

Comment:
the cleaner way to right this without needing to silence eslint is probably:
```
let body;
try {
  body = ...
} catch (err) {
    ...
}

```

Reply:
Style preference, var is much cleaner to me

Reply:
Is also exactly what var is made for 🙂

Reply:
You should avoid using `var` though and better stick with `let` in general. For that case it's even better to use `const`, since body isn't changing afterwards.

Reply:
You cannot use `const` here, because you have to hoist the value outside of the `try/catch` scope. The only (viable) options are to define `let` early & reassign, or to use `var` – which is it's designed & intended purpose. 

The eslint rule exists to prevent you from using `var` inline / in the same scope. In other words, it was made to prevent this kind of `var` usage:

```js
function hello(name, prefix) {
  var fullname = prefix + ' ' + name;
  console.log('Hello', fullname);
}
```

Reply:
I don't really care and am not going to nitpick on it too much, but I think a lot of what you're saying isn't true. `var` wasn't made specifically for the purpose of allowing access outside block scope. It preceded `let` and `const` which were introduced due to safety and readability issues with `var`. The eslint rule is meant to discourage use of `var` in its entirety: https://eslint.org/docs/rules/no-var#rule-details. If eslint wanted to allow the usage you were suggesting the rule certainly could have been written to do so.

But what's the better way to write this doesn't really matter as much as following what eslint is suggesting. If everyone committed code with their own individual style preferences the project would be a mess. The point of eslint and prettier is to enforce a consistent style and even if some people might disagree with aspects of it, it's still better that we have consistency than things written a bunch of different ways with `eslint-ignore` scattered throughout the code

Reply:
Agree it's an aside, but `var` has always behaved this way, by design. When `let` & `const` were introduced, it was a design choice to have both of them be block-scoped instead of being function-scoped.

Reply:
thanks for changing!